my-app/src/layouts/header/headerMenu/HeaderMenu.jsx

```diff
@@ -2,25 +2,22 @@ import './HeaderMenu.scss';
 
 import React from 'react';
 import {Link} from 'react-router-dom';
+import {BoardStore} from 'store/BoardStore';
 
-const Routs = {
```

Comment:
It's better to create folder like:

utils ---- folder
    constants ---- file
    helpers ---- file (with function that can help to develop)